pandas/tests/io/formats/test_console.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from pandas.io.formats.console import detect_console_encoding
+
+
+class MockEncoding(object):
+    """
+    TODO(py27): replace with mock
```

Comment:
I didn't see the LINT errors before but maybe you got it mixed up with the one I've commented on below? I still think inline with the `class` block opening would be preferable. Certainly a nit, but I haven't seen TODOs embedded in docstrings like this before